server/src/main/java/com/uj/projects/booksplatform/user/controller/UserController.java

```diff
@@ -42,6 +47,11 @@ public User registerUser(@Valid @RequestBody User user){
     @CrossOrigin(origins = "*", allowedHeaders = "*")
     @RequestMapping(value = "/resetPassword", method = RequestMethod.POST)
     public PasswordResetResponse ResetPassword(PasswordResetRequest request){
+        try {
+            passwordResetService.resetPassword(request.getEmail());
+        } catch (Exception | UserNotFoundException e) {
+            return new PasswordResetResponse(false, e.getMessage());
+        }
         return new PasswordResetResponse(true, "");
```

Comment:
why method always return success message without content ? Shouldn't it also give information about errors if they appear ?

Reply:
you can see line above, when it failed it returns exception message